packages/harmony/src/components/button/FilterButton/FilterButton.mdx

```diff
@@ -0,0 +1,54 @@
+import {
+  Meta,
+  Title,
+  Description,
+  Controls,
+  Canvas,
+  Primary
+} from '@storybook/blocks'
+
+import { Heading, RelatedComponents } from 'storybook/components'
+
+import * as FilterButton from './FilterButton.stories'
+
+<Meta of={FilterButton} />
+
+<Title />
+
```

Comment:
Should add a BetaWarning here

Reply:
yeah and to avoid the [beta] brackets and just have the "beta" component, update the title to be `<Title>Filter Button</Title>`

Reply:
yeah and to avoid the [beta] brackets and just have the "beta" component, update the title to be `<Title>Filter Button</Title>`

Reply:
how do i get beta to show in the nav then @dylanjeffers ?

Reply:
keep the name in the story, just ind the mdx you can override the title, dont need to do though if you dont want

Reply:
great sounds good!